Mathlib/NumberTheory/Padics/PadicNumbers.lean

```diff
@@ -609,16 +608,13 @@ theorem rat_dense' (q : ℚ_[p]) {ε : ℚ} (hε : 0 < ε) : ∃ r : ℚ, padicN
     ⟨q' N, by
       classical
       dsimp [padicNormE]
-      -- Porting note: `change` → `convert_to` (`change` times out!)
-      -- and add `PadicSeq p` type annotation
-      convert_to PadicSeq.norm (q' - const _ (q' N) : PadicSeq p) < ε
+      convert_to PadicSeq.norm (q' - const _ (q' N)) < ε
```

Comment:
Does `change` still time out here? If so, can we track it?

Reply:
Reverted.